src/assets/css/oag.css

```diff
@@ -0,0 +1,85 @@
+.oag-theme header .brand img, .oag-theme header .brand object {
+	max-height: none;
+}
+
+.oag-theme header .oag-gb {
+	background: #1a303c;
+}
+
+.oag-theme .btn-primary{
+	background-color: #803C2C;
+	border-color: #26374a;
+	border-radius: unset;
+}
+
+.oag-theme .btn-primary:hover {
+	background-color: #1A303C;
+	border-color: #091c2d;
+}
+
+.oag-theme .alert-info {
+	border-color: #7F3D2D;
+	-o-border-image: linear-gradient(to bottom, #7F3D2D 16px, #7F3D2D 16px, transparent 16px, transparent 48px, #7F3D2D 48px, #7F3D2D 48px) 1 100%;
+	border-image: linear-gradient(to bottom, #7F3D2D 16px, #7F3D2D 16px, transparent 16px, transparent 48px, #7F3D2D 48px, #7F3D2D 48px) 1 100%;
+	color: #676C72;
+}
+
+.oag-theme .alert-info::before {
+	color: #7F3D2D;
+	content: "";
```

Comment:
This line is not needed.

Reply:
It is used to override this css from the GCWeb theme.css:
.alert-info::before {
  color: #269abc;
}